HW2.1_Average.cpp

```diff
@@ -0,0 +1,19 @@
+#include <cstdint>
+#include <iostream>
+
+int main() {
+  std::cerr << "Please, enter number" << std::endl;
+  uint64_t number{0};
+  std::cin >> number;
+  int sum{0};
+  int counter{0};
+  for (; number != 0;) {
+    sum += (number % 10);
+    number /= 10;
+    ++counter;
+  }
+  std::cerr << "Here is the sum of the numbers: " << sum << '\n';
+  float average{static_cast<float>(sum) / static_cast<float>(counter)};
```

Comment:
Достаточно только один операнд скастить в вещественный тип, чтобы результат выражения был вещественным
```suggestion
  float average{static_cast<float>(sum) / counter};
```